projects/1.18/assets/dynamictrees/lang/zh_cn.json

```
@@ -0,0 +1,119 @@
+
+  "block.dynamictrees.oak_branch": "有活力的橡木",
+  "block.dynamictrees.azalea_branch": "有活力的杜鹃木",
+  "block.dynamictrees.spruce_branch": "有活力的云杉木",
+  "block.dynamictrees.birch_branch": "有活力的白桦木",
+  "block.dynamictrees.jungle_branch": "有活力的丛林木",
+  "block.dynamictrees.dark_oak_branch": "有活力的深色橡木",
+  "block.dynamictrees.acacia_branch": "有活力的金合欢木",
+  "block.dynamictrees.crimson_branch": "有活力的绯红菌柄",
+  "block.dynamictrees.warped_branch": "有活力的诡异菌柄",
+  "block.dynamictrees.rooty_water": "含水树根",
+
+  "species.dynamictrees.oak": "橡树",
+  "species.dynamictrees.azalea": "杜鹃树",
+  "species.dynamictrees.swamp_oak": "沼泽橡树",
+  "species.dynamictrees.spruce": "云杉",
+  "species.dynamictrees.mega_spruce": "大型云杉",
+  "species.dynamictrees.birch": "白桦",
+  "species.dynamictrees.tall_birch": "高白桦",
+  "species.dynamictrees.jungle": "丛林木",
+  "species.dynamictrees.mega_jungle": "大型丛林木",
+  "species.dynamictrees.dark_oak": "深色橡树",
+  "species.dynamictrees.acacia": "金合欢",
+  "species.dynamictrees.apple_oak": "苹果树",
+  "species.dynamictrees.crimson": "绯红菌树",
+  "species.dynamictrees.mega_crimson": "大型绯红菌树",
+  "species.dynamictrees.warped": "诡异菌树",
+  "species.dynamictrees.mega_warped": "大型诡异菌树",
+  "species.dynamictrees.red_mushroom": "红色蘑菇",
+  "species.dynamictrees.brown_mushroom": "棕色蘑菇",
+  "species.dynamictrees.cocoa": "可可树",
+  "species.dynamictrees.oak_undergrowth": "橡树灌木",
+  "species.dynamictrees.jungle_undergrowth": "丛林木灌木",
+
+  "item.dynamictrees.oak_seed": "橡子",
+  "item.dynamictrees.azalea_seed": "杜鹃种子",
+  "item.dynamictrees.spruce_seed": "云杉球果",
+  "item.dynamictrees.birch_seed": "白桦絮",
+  "item.dynamictrees.jungle_seed": "丛林木荚",
+  "item.dynamictrees.cocoa_seed": "可可果荚",
+  "item.dynamictrees.dark_oak_seed": "深色橡子",
+  "item.dynamictrees.acacia_seed": "金合欢种子",
+  "item.dynamictrees.apple_oak_seed": "苹果仁",
+  "item.dynamictrees.crimson_seed": "绯红菌子实体",
+  "item.dynamictrees.warped_seed": "诡异菌子实体",
+
+  "item.dynamictrees.dendro_potion.biochar": "木炭溶液",
+  "item.dynamictrees.dendro_potion.depletion": "枯竭药水",
+  "item.dynamictrees.dendro_potion.gigas": "放大药水",
+  "item.dynamictrees.dendro_potion.burgeoning": "生长药水",
+  "item.dynamictrees.dendro_potion.fertility": "肥沃药水",
+  "item.dynamictrees.dendro_potion.persistence": "定型药水",
+  "item.dynamictrees.dendro_potion.transform": "转生药水",
+  "item.dynamictrees.dendro_potion.harvest": "丰收药水",
+  "item.dynamictrees.dendro_potion.denuding": "剥落药水",
+
+  "item.dynamictrees.dirt_bucket": "泥桶",
+  "item.dynamictrees.staff": "森林之杖",
+  "block.dynamictrees.potted_sapling": "被种下的树苗",
+
+  "chat.registry_name": "注册表名称：%s",
+
+  "tooltip.dynamictrees.species": "树种：%s",
+  "tooltip.dynamictrees.fertility": "土壤肥力：%s",
+  "tooltip.dynamictrees.jo_code": "JoCode：%s",
+  "tooltip.dynamictrees.force_planting": "强制种植：%s",
+  "tooltip.dynamictrees.seed_life_span": "种子寿命：%s",
+
+  "tooltip.dynamictrees.enabled": "已启用",
+
+  "commands.dynamictrees.success.get_tree": "树种：%s的 JoCode 为：%s。",
```

```suggestion
  "commands.dynamictrees.success.get_tree": "树种：%s的JoCode为：%s。",
```
1.16开始所有版本不再需要在半角字符和全角字符之间留下空格

---

了解，就这样改吧